src/boot/CIC6105.c

```diff
@@ -2,23 +2,65 @@
 
 #include "build.h"
 #include "fault.h"
+#include "macros.h"
+#include "regs.h"
+#include "scheduler.h"
 
-#pragma increment_block_number "n64-us:200"
+#pragma increment_block_number "n64-us:188"
+
+// TODO: Consider moving to a header file.
+extern u64 cic6105TextStart[];
+extern u64 cic6105TextEnd[];
 
 s32 gCICAddr1Val;
 s32 gCICAddr2Val;
 FaultClient sRomInfoFaultClient;
+#if MM_VERSION <= N64_JP_1_1
+s32 gCICValue1;
+s32 gCICValue2;
+
+s32 CIC6105_ILoveYou(void);
+#endif
 
 void CIC6105_Noop1(void) {
+#if MM_VERSION < N64_US
```

Comment:
Not quite a Noop anymore. We can probably figure out the context from OoT, but it is fine if you want to keep it Noop for now.